Marlin/src/module/delta.cpp

```diff
@@ -38,7 +38,8 @@
 #include "../Marlin.h"
 
 // Initialized by settings.load()
-float delta_height,
+float G33_offset,
```

Comment:
You should not define this new 4-byte global that steals SRAM unless it is used.
So please wrap it in `#if ENABLED(DELTA_AUTO_CALIBRATION)`

Reply:
Trade off:
- 4 bytes of extra SRAM for DELTA && !DELTA_AUTO_CALIBRATION in 1 single unused variable
- xxx-bytes of extra SRAM for  DELTA && !DELTA_AUTO_CALIBRATION in extra code in EEPROM- write, -read, -reset,...

Can't have it both, so choose whatever you prefer.

Reply:
You're right, something should be done for delta users who do not use G33 to calibrate as well, we can't leave them out in the cold. See additional commit.